integration/service/src/main/java/com/cerner/beadledom/integration/service/resource/HelloWorldResource.java

```diff
@@ -0,0 +1,43 @@
+package com.cerner.beadledom.integration.service.resource;
+
+import com.cerner.beadledom.integration.api.model.HelloWorldDto;
+import com.cerner.beadledom.jaxrs.GenericResponse;
+import com.cerner.beadledom.jaxrs.GenericResponses;
+import com.cerner.beadledom.pagination.OffsetPaginatedList;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import java.util.ArrayList;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Server implementation of an offset based paginated list endpoint.
+ *
+ * @author Nick Behrens
+ */
+@Api(value = "/hello", description = "Retrieve hello world data")
+@Path("/hello")
+public class HelloWorldResource {
+
+  /**
+   * An endpoint that returns a paginated list of {@link HelloWorldDto}.
+   *
+   * @return an {@link OffsetPaginatedList} containing {@link HelloWorldDto}'s.
+   */
+  @ApiOperation(
+      value = "Retrieves hello world data.",
+      response = HelloWorldDto.class)
```

Comment:
Shouldn't response here be OffsetPaginatedListHelloWorldDto or OffsetPaginatedList<HelloWorldDto>?

Reply:
It ideally should be `OffsetPaginatedList<HelloWorldDto>` but swagger annotations don't support generics and we can't use the built in support because we are actually returning a `GenericResponse<OffsetPaginatedList<HelloWorldDto>>` which is way messier of a swagger model then just a HelloWorldDto is.

Reply:
Then shouldn't it just be OffsetPaginatedList ?

Reply:
So with the way the swagger annotations work, generics aren't represented at all. So putting `OffsetPaginatedList<HelloWorldDto>` in there is out of the question. I would prefer it to be a `HelloWorldDto` because thats the actual object as opposed to just the list interface that is wrapping it.